sdk/eventgrid/Azure.Messaging.EventGrid/tests/SessionRecords/EventGridClientTests/CanPublishEvent.json

```diff
@@ -1,36 +1,36 @@
 {
   "Entries": [
     {
-      "RequestUri": "https://t-kerlee-customtopic.westus-1.eventgrid.azure.net/api/events?api-version=2018-01-01",
+      "RequestUri": "https://t-kerlee-topic.westus-1.eventgrid.azure.net/api/events?api-version=2018-01-01",
```

Comment:
nit: was there a reason to re-record this test? It's fine to leave it, but generally we only re-record tests if it is necessary, for instance if we are adding a new header to the request.

Reply:
No reason - I think I just accidentally re-ran all of the tests with a new topic.